functions-and-scope.js

```diff
@@ -28,6 +53,20 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // cumLaude([8, 9, 4, 6, 10]) geeft 3
 
 
+function cumLaude( grades ) {
+let amount = 0
+    for( let i = 0; i < grades.length; i++ ) {
+        if (grades [ i ] >= 8 ) amount++
+        values = grades [  i ]
+        amount = amount + values
```

Comment:
Je hoeft alleen te tellen hoeveel cijfers hoger of gelijk aan een 8 zijn, je kunt dus deze twee regels weglaten